### PR TITLE
Added some test in e2e for verification

### LINKED ISSUE #426: SOHO-7388: .beforeActivate and .activated event is fired for every tab when closing tab
**Describe the bug**
Every time a tab is closed (opened) events .beforeActivate and .activated are fired for **every** opened tab. Expected behavior is to fire this event for specific tab that is deactivated and the one that is activated.

**To Reproduce**
1. Go to http://master-enterprise.demo.design.infor.com/components/tabs/example-activated-event.html
2. Edit the example to make it more clear (by putting the events in the console
2. Click a different tab

**Expected behavior**
Expect one event for deactivate for the tab that is deactivated and one for the tab that is activated.

**Platform**
All

**Additional context**
See SOHO-7388

NOTE: I am not seeing this issue on http://master-enterprise.demo.design.infor.com/components/tabs/example-activated-event.html

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added some additional e2e spec test just to cover some verification of the callbacks

**Related github/jira issue (required)**:

Closes #426 

**Steps necessary to review your pull request (required)**:

### CODE REVIEW

File: test/components/tabs/tabs.e2e-spec.js [hunk 1]
```diff
@@ -477,7 +477,7 @@ describe('Tabs ajax as href tests', () => {
     await utils.checkForErrors();
   });
 
-  xit('Should be able to activate href tabs', async () => { //eslint-disable-line
+  it('Should be able to activate href tabs', async () => { //eslint-disable-line
```
Comment (tmcconechy) [inline test/components/tabs/tabs.e2e-spec.js]: Can remove //eslint-disable-line now.

File: test/components/tabs/tabs.e2e-spec.js [hunk 2]
```diff
@@ -477,7 +477,7 @@ describe('Tabs ajax as href tests', () => {
...
     // Cant get this test to work on CI anymore
```
Comment (tmcconechy) [inline test/components/tabs/tabs.e2e-spec.js]: I guess this isnt true anymore, can remove the comment?

File: test/components/tabs/tabs.e2e-spec.js [hunk 3]
```diff
@@ -491,3 +491,36 @@ describe('Tabs ajax as href tests', () => {
...
+    const tabsContainerEl = await element(by.id('tabs'));
+    await browser.driver
+      .wait(protractor.ExpectedConditions.presenceOf(tabsContainerEl), config.waitsFor);
+  });
+
+  it('Should not have errors', async () => {
+    await utils.checkForErrors();
+  });
+
+  it('Should be able to call activate and beforeActivate in tabs', async () => { //eslint-disable-line
```
Comment (tmcconechy) [inline test/components/tabs/tabs.e2e-spec.js]: can remove eslint-dsable-line

File: test/components/tabs/tabs.e2e-spec.js [hunk 4]
```diff
@@ -491,3 +491,36 @@ describe('Tabs ajax as href tests', () => {
...
+
+    await browser.actions().mouseMove(opportunitiesEl).perform();
+    await browser.actions().click(opportunitiesEl).perform();
+    await opportunitiesEl.click();
+
+    const beforeActivateEl = await element.all(by.css('.toast-title')).get(0);
+    await browser.driver
+      .wait(protractor.ExpectedConditions.presenceOf(beforeActivateEl), config.waitsFor);
+
+    expect(beforeActivateEl.getText()).toContain('beforeActivate');
```
Comment (tmcconechy) [inline test/components/tabs/tabs.e2e-spec.js]: await or requery here.

File: test/components/tabs/tabs.e2e-spec.js [hunk 5]
```diff
@@ -491,3 +491,36 @@ describe('Tabs ajax as href tests', () => {
...
+
+    const activateEl = await element.all(by.css('.toast-title')).get(1);
+    await browser.driver
+      .wait(protractor.ExpectedConditions.presenceOf(activateEl), config.waitsFor);
+
+    expect(activateEl.getText()).toContain('activated');
```
Comment (tmcconechy) [inline test/components/tabs/tabs.e2e-spec.js]: await or requery here.

**Comment by tjamesallen15** [comment]
Updated the file of e2e.

**Pull request merged**